src/Sparrow/project.json

```diff
@@ -20,10 +20,17 @@
         "optimize": true,
         "define": [ "MEM_GUARD" ]
       }
+    },
+    "Validate": {
+      "buildOptions": {
+        "optimize": true,
+        "define": [ "MEM_GUARD", "MEM_GUARD_STACK" ]
+      }
     }
   },
 
   "dependencies": {
+    "DotNetCross.Memory.Unsafe": "0.2.3.4",
```

Comment:
How much of a perf boost does this give us, and do we need to modify additional places?